charm/docs/tutorial/getting-started.md

```diff
@@ -34,7 +37,8 @@ juju add-model netbox-tutorial
 Deploy the NetBox charm, with all its mandatory requirements (PostgreSQL, Redis and S3).
 
 ### Deploy the charms:
```

Comment:
I would update this heading to say
`## Deploy the charms`

Reply:
I have changed it a bit, as it was badly ordered..